hooks/pre-commit

```diff
@@ -1,25 +1,27 @@
-#!/bin/bash
-if clang-format --version | grep -q 'version 11\.'; then
-   CLANG_FORMAT_EXECUTABLE="clang-format"
-else
-   CLANG_FORMAT_EXECUTABLE="clang-format-11"
-fi
+#!/bin/sh
+
+name=""
+version="14.0.0"
```

Comment:
Make this variable contain the minimum accepted version, and don't modify it in the script.

```suggestion
minVersion="14.0.0"
```